doc/nrf/libraries/modem/location.rst

```diff
@@ -143,6 +142,14 @@ The following option is useful when setting :kconfig:option:`CONFIG_NRF_CLOUD_AG
 
 * :kconfig:option:`CONFIG_NRF_CLOUD_AGPS_ELEVATION_MASK` - Sets elevation threshold angle.
 
+The obstructed visibility feature is based on the fact that the number of satellites found indoors or in other environments with limited sky-view is severely decreased.
+The following options control the sensitivity of obstructed visibility detection.
```

Comment:
Yep, the sentence "The following options..." would be better placed just before the listed options. But this would mean some rewording and reordering of the other sentences.

Maybe the one "They set the..." could be after the listed options and begin with "These options set the..."